src/box/rt_lua_call_access.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <zlib.h>
+#include <lua.h>
+#include <lauxlib.h>
+#include <lualib.h>
```

Comment:
`zlib` is not used here, Lua function too. And there are no booleans. Please, drop the unused includes.

Also, I see that the `.c` file doesn't use anything like `diag_set`, so `diag.h` include is redundant too.

```diff
diff --git a/src/box/rt_lua_call_access.c b/src/box/rt_lua_call_access.c
index 0fc486bf83..1bf4904713 100644
--- a/src/box/rt_lua_call_access.c
+++ b/src/box/rt_lua_call_access.c
@@ -1,5 +1,4 @@
 #include "rt_lua_call_access.h"
-#include "diag.h"
 #include "assoc.h"
 #include "user.h"
 #include "session.h"
diff --git a/src/box/rt_lua_call_access.h b/src/box/rt_lua_call_access.h
index f5fc371aec..c427f29c11 100644
--- a/src/box/rt_lua_call_access.h
+++ b/src/box/rt_lua_call_access.h
@@ -1,11 +1,6 @@
 #pragma once
 
 #include <stdint.h>
-#include <stdbool.h>
-#include <zlib.h>
-#include <lua.h>
-#include <lauxlib.h>
-#include <lualib.h>
 
 #if defined(__cplusplus)
 extern "C" {
```

Reply:
Done.